.github/workflows/documentation_on_tags.yml

```
@@ -0,0 +1,53 @@
+name: Build stable documentation
+
+on:
+  push:
+    tags:
+      - '*'
```

```suggestion
      - "*.*.*"
```

As we follow semver, every tag should look like this